src/docs/asciidoc/40-about.adoc

```diff
@@ -0,0 +1,62 @@
+== About This Project
+
+=== Maintainers
+
+* https://github.com/cdancy[Christopher Dancy]
+* https://github.com/bmuschko[Benjamin Muschko]
+* https://github.com/orzeh[Orzeh]
```

Comment:
@orzeh Let me know if you'd like your real name here. Your GitHub account doesn't reflect it.

Reply:
why not. My name is Łukasz Warchał

Reply:
done